April webinar (#589)
flowforge/website: src/webinars/2023/industrial-data-node-red.md

---

> title: "Connect, Integrate, Visual Industrial Production Metrics with Node-RED"

Reviewer: ```suggestion
title: "Connect, Integrate & Visualize Industrial Production Metrics with Node-RED"
```

Sentence structure didnt feel right (though I couldn't put my finger on why)

Open to debate.

subtitle: Join FlowForge's Steve McLaughlin as he showcases how easy it is to use Node-RED to visualize popular production metrics using Node-RED.
image: /images/webinars/webinar-april.jpg
date: 2023-04-27
time: 17:00 CEST (11:00 ET) 
duration: 60

> hosts: ["stephe-mclaughlin"]

Reviewer: ```suggestion
hosts: ["stephen-mclaughlin"]
```

hubspot:
    formId: b4ec9aa1-8648-47b3-b908-ec8597e8735d
---

**Node-RED makes it very easy to connect industrial equipment so that data can be visualized to understand common production metrics like OEE, performance, cycle times etc.**

<!--more-->

In this webinar, attendees will learn how to create a Node-RED flow to:
- Connect to an existing PLC using an industrial protocol
- Integrate data from an external MQTT broker
- Create a dashboard to visualize common production metrics

> - Filter the data to report by exception and store this data in a database.

Reviewer: ```suggestion
- Sanitize data for recording in a database.
- Generate a report from recorded data
```

This webinar will include a live demonstration of how to set up Node-RED to accomplish the tasks listed above. Attendees are expected to have a general understanding of Node-RED.

To attend the webinar please register in advance.

**About Steve:**

> Steve McLaughlin, Node-RED OSS Developer at FlowForge, will be leading this webinar. Steve has over 25 years as an engineer in the automotive sector primarily interfacing field technologies and IS/IT. During his work in the automotive industry, he discovered the power of using Node-RED to connect, integrate and visual industrial data. Steve is also a long time member of the Node-RED community.

Reviewer: ```suggestion
Steve McLaughlin, Node-RED OSS Developer at FlowForge, will be leading this webinar. Steve has over 25 years as an engineer in the automotive sector primarily interfacing field technologies and IS/IT. During his work in the automotive industry, he discovered the power of using Node-RED to connect, integrate and visualize industrial data. Steve is also a long time member of the Node-RED community.
```

---
title: "Connect, Integrate & Visualize Industrial Production Metrics with Node-RED"
subtitle: Join FlowForge's Steve McLaughlin as he showcases how easy it is to use Node-RED to visualize popular production metrics using Node-RED.
image: /images/webinars/webinar-april.jpg
date: 2023-04-27
time: 17:00 CEST (11:00 ET) 
duration: 60
hosts: ["stephen-mclaughlin"]
hubspot:
    formId: b4ec9aa1-8648-47b3-b908-ec8597e8735d
---

**Node-RED makes it very easy to connect industrial equipment so that data can be visualized to understand common production metrics like OEE, performance, cycle times etc.**

<!--more-->

In this webinar, attendees will learn how to create a Node-RED flow to:
- Connect to an existing PLC using an industrial protocol
- Integrate data from an external MQTT broker
- Create a dashboard to visualize common production metrics

> - Sanitize data for recording in a database.

Reviewer: ```suggestion
- Sanitize data for recording in a database
```

---
title: "Connect, Integrate, Visual Industrial Production Metrics with Node-RED"
subtitle: Join FlowForge's Steve McLaughlin as he showcases how easy it is to use Node-RED to visualize popular production metrics using Node-RED.
image: /images/webinars/webinar-intro-ff.jpg
date: 2023-04-27
time: 17:00 CEST (11:00 ET) 
duration: 60
hosts: ["steve-mclaughlin"]
hubspot:
    formId: b4ec9aa1-8648-47b3-b908-ec8597e8735d
---

**Node-RED makes it very easy to connect industrial equipment so that data can be visualized to understand common production metrics like OEE, performance, cycle times etc. **

<!--more-->

In this webinar, attendees will learn how to create a Node-RED flow to:

> - Connect to an existing PLC using Modbus

Reviewer: ```suggestion
- Connect to an existing PLC using Industrial Protocols
```

I dont currently have a ModBus capable device. 

I can of course buy something (cheap and visual) if we must use ModBus.

> - Integrate data from an external source

Reviewer: _comment only_

We can very easily grab data from one of the many open public MQTT brokers

Or look around for a web API like weather (though I dont think there are any free ones left!)

Reviewer: By using Public MQTT brokers, people can "follow along"

Also, MQTT is very (I)IoT flavour

Author: OK, I will change to be Integrate data from an MQTT broker.

> - Create a dashboard to visualize common production metrics

Reviewer: _comment only_

This can be a little time consuming so I would propose we demonstrate setup of 1 or 2 UI elements and then have a "look what we made earlier" importable set of dashboard nodes.

> - Filter the data to report by exception and store this data in a [InfluxDB] database.

Reviewer: Must we use Influx? 
Is it your thoughts and intention we will be displaying the Time Series data in Graphana?

My personal thoughts were heading to to MS-SQL (as it is _very corporate_) OR PostGRES and have PDF reports generated from the data.

Author: MS-SQL if fine. I will update.
